.prettierrc

```diff
@@ -1,5 +1,10 @@
 {
-  "singleQuote": true,
+  "arrowParens": "avoid",
   "bracketSpacing": false,
-  "quoteProps": "consistent"
+  "printWidth": 120,
+  "quoteProps": "consistent",
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "none"
```

Comment:
This will obviously change a lot of files, but I'm a _big_ fan of trailing commas. It cleans up diffs so much.

Reply:
My thinking on this is I don’t have to stare at diffs all day long, in the code they stick out like a sore thumb to me 😄 

Other people have opinions on this?

Reply:
It _is_ an acquired taste for sure!

Reply:
It does help, like @JoeyBG said, to keep diffs at a minimum when adding an element to the list at the end. But I do not care really, and always remove it anyway.

Reply:
After an internal discussion we decided to follow Prettier’s decision to enforce trailing commas